test/options_test.rb

```diff
@@ -88,5 +88,9 @@ module Slop; class BarOption < Option; end; end
       @options_config = Slop::Options.new({banner: "custom banner"})
       assert_match(/^custom banner/, @options_config.to_s) 
     end
+    it "banner is disabled" do
+      @options_config = Slop::Options.new({banner: false})
+      assert_match(//, @options_config.to_s)
```

Comment:
Yep, I think you could probably just use `""` instead of a regular expression, though. Could do that and then squash your commits into one? Happy to merge then :)